extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile

```diff
@@ -6,9 +6,9 @@ ARG DELTA_HIVE_CONNECTOR_VERSION
 LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 
 RUN curl -L https://github.com/delta-io/connectors/releases/download/v${DELTA_HIVE_CONNECTOR_VERSION}/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar \
-          -o "${SPARK_HOME}"/jars/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar
+          -o "$SPARK_HOME"/jars/delta-hive-assembly_2.12-${DELTA_HIVE_CONNECTOR_VERSION}.jar
 ARG JARS_DIR
-ADD ${JARS_DIR}/* ${SPARK_HOME}/jars/
+ADD ${JARS_DIR}/* $SPARK_HOME/jars/
```

Comment:
What about `${JARS_DIR}`?

Reply:
The build argument can be replaced with ${}

Reply:
How is this any different then the `$JARS_DIR` in `extensions/extensions-docker/aissemble-hive/aissemble-hive-service/src/main/resources/docker/Dockerfile`?